OrbitGl/TimeGraph.cpp

```diff
@@ -593,8 +612,10 @@ void TimeGraph::UpdatePrimitives(bool a_Picking) {
               } else if( timer.m_Type == Timer::INTROSPECTION && string_manager_) {
                 textBox.SetText(string_manager_->Get(
                     timer.m_UserData[0]).value_or(""));
-              }
-               else if (!SystraceManager::Get().IsEmpty()) {
+              } else if ( timer.m_Type == Timer::GPU_ACTIVITY && string_manager_) {
```

Comment:
Can string_manager_ be nullptr? if not, maybe add CHECK above and drop these

Reply:
Added CHECK at the top of this method.